drivers/sdhc/Kconfig.stm32

```
@@ -0,0 +1,23 @@
+# Copyright (c) 2025 EXALT Technologies.
+# SPDX-License-Identifier: Apache-2.0
+
+source "subsys/logging/Kconfig.template.log_config"
+
+config SDHC_STM32_SDIO
+	bool "STM32 SDMMC driver support for SDIO"
+	depends on DT_HAS_ST_STM32_SDIO_ENABLED
+	select USE_STM32_HAL_SDIO
+	select USE_STM32_LL_SDMMC
+	select SDHC_SUPPORTS_NATIVE_MODE
+	imply CACHE_MANAGEMENT if !SDHC_STM32_POLLING_SUPPORT && DCACHE
+	default y
+	help
+	  Enable support for SDMMC on STM32 platforms.
```

Sorry, I missed this one:
```suggestion
	  Enable support for SDMMC on STM32 platforms for SDIO devices.